src/framework/global/tests/string_tests.cpp

```diff
@@ -53,6 +53,9 @@ TEST_F(Global_Types_StringTests, Char_Digit)
         EXPECT_FALSE(ch.isDigit());
         EXPECT_EQ(ch.digitValue(), -1);
     }
+
+    QString str = "ggg";
+    bool ret = str >= 7;
```

Comment:
Looks like this is either incomplete or a left-over :)

Reply:
oops )